docs/oracle/umips.md

```diff
@@ -0,0 +1,12 @@
+---

```

Comment:
How can we not duplicate content in various sections of the doc site?

Most of this is here:

https://docs.umaproject.org/uma-tokenholders/umips

Reply:
Although there is overlap, that link is quite detailed, the idea of this section was to give a general overview of what an UMIP was.  Maybe add link to the end for further info?